code/__DEFINES/conflict.dm

```diff
@@ -18,24 +18,32 @@
 #define AMMO_ENERGY (1<<4)
 #define AMMO_ROCKET (1<<5)
 #define AMMO_SNIPER (1<<6)
-#define AMMO_ANTISTRUCT (1<<7) // Primarily for railgun but can be implemented for other projectiles that are for antitank and antistructure (wall/machine)
+
+/// Primarily for railgun but can be implemented for other projectiles that are for antitank and antistructure (wall/machine)
+#define AMMO_ANTISTRUCT (1<<7)
+
 #define AMMO_SKIPS_ALIENS (1<<8)
 #define AMMO_IGNORE_ARMOR (1<<9)
 #define AMMO_IGNORE_RESIST (1<<10)
 #define AMMO_BALLISTIC (1<<11)
 #define AMMO_IGNORE_COVER (1<<12)
-//   (1<<13) unused, previously was AMMO_SCANS_NEARBY
+//                              (1<<13) unused, previously was AMMO_SCANS_NEARBY
 #define AMMO_STOPPED_BY_COVER (1<<14)
 #define AMMO_SPECIAL_EMBED (1<<15)
-#define AMMO_STRIKES_SURFACE (1<<16) // If the projectile hits a dense turf it'll do on_hit_turf on the turf just in front of the turf instead of on the turf itself
-#define AMMO_HITS_TARGET_TURF (1<<17) // Whether or not the bullet hits the target that was clicked or if it keeps travelling
+
+/// If the projectile hits a dense turf it'll do on_hit_turf on the turf just in front of the turf instead of on the turf itself
+#define AMMO_STRIKES_SURFACE (1<<16)
+/// Whether or not the bullet hits the target that was clicked or if it keeps travelling
+#define AMMO_HITS_TARGET_TURF (1<<17)
+
```

Comment:
```suggestion
/// If the projectile hits a dense turf it'll do on_hit_turf on the turf just in front of the turf instead of on the turf itself
#define AMMO_STRIKES_SURFACE (1<<16)
/// Whether or not the bullet hits the target that was clicked or if it keeps travelling
#define AMMO_HITS_TARGET_TURF (1<<17)
```